app/pages/account.tsx

```diff
@@ -159,6 +159,8 @@ export const AccountPage = ({ user, constructorPage, prefs }: AuthProps) => {
             ''
           )}
           is displayed next to any comments you make or puzzles you create.
+          
+          If set, it will also be used in the pause/resume modal on each puzzle&apos;s page instead of your email address.
```

Comment:
Wasn't sure how to call out this part of the app. Opted for the language used by the initial issue's reporter.

Reply:
My gut is to just remove this note entirely. I feel that the line above it is important because the display name appearing next to comments/puzzles is a *public* appearance so users should be aware of where it will appear to others. For this usage of it it's only seen by the user in question so I think it's fine to omit here in the interest of brevity. Thoughts?

Reply:
That makes sense to me. I've removed it.